0225AH/data/0213IbnHisham/0213IbnHisham.SiraNabawiyya/Datasets/0213IbnHisham.SiraNabawiyya.Shamela0023833-ara1_superheadings.csv

```diff
@@ -1,2375 +1,2374 @@
+3,,### || (محاربة قصي لخزاعة وبني بكر وتحكيم يعمر بن عوف) :,34.2.0
+3,,### || (سبب تسمية يعمر بالشداخ) :,34.3.0
+3,,### || (قصي أميرا على مكة وسبب تسميته مجمعا) :,34.4.0
+3,,### || (شعر رزاح في نصرته قصيا ورد قصي عليه) :,34.5.0
+3,,"### || (ما كان بين رزاح وبين نهد وحوتكة، وشعر قصي في ذلك) :",34.6.0
+3,,### || (ما آثر به قصي عبد Milestone300 الدار) :,34.7.0
+3,,### || (الرفادة) :,34.8.0
+3,,### | ذكر ما جرى من اختلاف قريش بعد قصي وحلف المطيبين,35.0.0
+3,,### || (الخلاف بين بني عبد الدار وبني أعمامهم) :,35.1.0
+3,,"### || (من ناصروا بني عبد الدار، ومن ناصروا بني أعمامهم.) :",35.2.0
+3,,### || (من دخلوا في حلف المطيبين) :,35.3.0
+3,,### || (من دخلوا في حلف الأحلاف) :,35.4.0
+3,,### || (توزيع القبائل في الحرب) :,35.5.0
+3,,### || (ما تصالح القوم عليه) :,35.6.0
+3,,### | حلف الفضول,36.0.0
+3,,### || (سبب تسميته كذلك) :,36.1.0
+3,,### || (حديث رسول الله صلى الله عليه وسلم عن حلف الفضول) :,36.2.0
+3,,"### || (نازع الحسين الوليد في حق، وهدد بالدعوة إلى حلف الفضول) :",36.3.0
+3,,"### || (سأل عبد الملك محمد بن جبير عن عبد شمس وبني نوفل ودخولهما في حلف الفضول، فأخبره بخروجهما منه) :",36.4.0
+3,,### || (ولاية هاشم الرفادة والسقاية وما كان يصنع إذا قدم الحاج) :,36.5.0
+3,,### || (شيء من أعمال هاشم) :,36.6.0
+3,,### || (ولاية المطلب الرفادة والسقاية) :,36.7.0
+3,,### || (زواج هاشم) :,36.8.0
+3,,### || (ميلاد عبد المطلب وسبب تسميته كذلك) :,36.9.0
+3,,### || (موت المطلب وما قيل في رثائه من الشعر) :,36.10.0
+3,,### || (ولاية عبد المطلب السقاية والرفادة) :,36.11.0
+3,,### | ذكر حفر زمزم وما جرى من الخلف فيها,37.0.0
+3,,### || (الرؤيا التي أريها عبد المطلب في حفر زمزم) :,37.1.0
+3,,### || (عبد المطلب وابنه الحارث وما كان بينهما وبين قريش عند حفرهما زمزم) :,37.2.0
+3,,### | ذكر بئار قبائل قريش بمكة,38.0.0
+3,,### || (الطوي ومن حفرها) :,38.1.0
+3,,### || (بذر ومن حفرها) :,38.2.0
+3,,### || (سجلة ومن حفرها) :,38.3.0
+3,,### || (الحفر ومن حفرها) :,38.4.0
+3,,### || (سقية ومن حفرها) :,38.5.0
+3,,### || (أم أحراد ومن حفرها) :,38.6.0
+3,,### || (السنبلة ومن حفرها) :,38.7.0
+3,,### || (الغمر ومن حفرها) :,38.8.0
+3,,### || (رم وخم والحفر وأصحابها) :,38.9.0
+3,,### || (فضل زمزم وما قيل فيها من شعر) :,38.10.0
+3,,### | ذكر نذر عبد المطلب ذبح ولده,39.0.0
+3,,### || (الضرب بالقداح عند العرب) :,39.1.0
+3,,### || (عبد المطلب وأولاده بين يدي صاحب القداح) :,39.2.0
+3,,"### || (خروج القدح على عبد الله وشروع أبيه في ذبحه، ومنع قريش له) :",39.3.0
+3,,### || (عرافة الحجاز وما أشارت به أم عبد المطلب) :,39.4.0
+3,,### || (نجاة عبد الله من الذبح) :,39.5.0
+3,,### | ذكر المرأة المتعرضة لنكاح عبد الله بن عبد المطلب,40.0.0
+3,,### || (رفض عبد الله طلب المرأة التي عرضت نفسها عليه) :,40.1.0
+3,,### || (زواج عبد الله من آمنة بنت وهب) :,40.2.0
+3,,### || (أمهات آمنة بنت وهب) :,40.3.0
+3,,### || (ما جرى بين عبد الله والمرأة المتعرضة له بعد بنائه بآمنة) :,40.4.0
+3,,### | ذكر ما قيل لآمنة عند حملها برسول الله صلى الله عليه وسلم,41.0.0
+3,,### || (موت عبد الله) :,41.1.0
+4,ذكر ولادة رسول الله ص وسيرته قبل مبعثه,### | ولادة رسول الله صلى الله عليه وسلم ورضاعته,42.0.0

```

Comment:
This line is referenced as `43.0.0` in the storyline data:

https://github.com/eldarion-client/DSP-Data/blob/59aff111d677c1a518bbb85b3dff99ba3ee49fb5/global%20datasets/storyline/storyline_novols.csv#L26